tiltIndicator_tiltIndicatorAfter_workflow.Rmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "tiltInidicator_tiltIndicatorAfter_workflow"
+author: "Kalash"
+date: "2023-07-21"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+The tiltIndicatorAfter_runner script uses the output from the tiltIndicatorBefore package as the inputs to the tiltIndicator and tiltIndicatorAfter packages. This script uses the CSV files from the tiltIndicatorBefore package to create the final output of the four indicators. 
+
+## Setup
+```{r}
+library(dplyr, warn.conflicts = FALSE)
+library(readr, warn.conflicts = FALSE)
+library(tidyr, warn.conflicts = FALSE)
+devtools::install_github("2DegreesInvesting/tiltIndicator")
```

Comment:
- [ ] This must be commented out. The users will need to install the package only once, not every time they run this file. Also installing packages is a pretty invasive action since it modifies the user's system. Thus it's best practice to just show the users how to do it but then it's up to them to do it.